packages/pds/src/actor-store/repo/sql-repo-reader.ts

```diff
@@ -136,6 +137,14 @@ export class SqlRepoReader extends ReadableBlockstore {
     return builder.execute()
   }
 
+  async countBlocks(): Promise<number> {
```

Comment:
Just a note that this could cause some blocking.

Reply:
Since it's only used for reads, I don't think it would block, but it might fail with a `SQLITE_BUSY` and have to retry

Reply:
Ah sorry didn't explain that well!  I was picturing blocking the main app thread.  E.g. if there's a second of work in sqlite to do here, the app thread doing the work will block for a second.